params/config.go

```diff
@@ -185,6 +191,11 @@ type XDPoSConfig struct {
 	FoudationWalletAddr    common.Address `json:"foudationWalletAddr"` // Foundation Address Wallet
 	SkipValidation         bool           //Skip Block Validation for testing purpose
 	V2ConsensusBlockNumber *big.Int
+	ConsensusV2Config      ConsensusV2Config
```

Comment:
maybe remove `Consensus` on `V2ConsensusBlockNumber` and `ConsensusV2Config`

Reply:
done